src/_layouts/base.html

```diff
@@ -159,27 +159,20 @@
     </div>
 </div>
 
-<div class="sliding-nav">
-    {% block primary_nav %}
-        {% include 'templates/nav/primary-nav.html' %}
-    {% endblock primary_nav %}
-
-    <div class="sliding-nav_page">
-    {% block header %}
-        {% include 'templates/header.html' %}
-    {% endblock header %}
-
-    <!-- PRIMARY CONTENT -->
-    {% block content %}
-        This will be replaced in templates that extend this template and override "content".
-    {% endblock content %}
-     <!-- /PRIMARY CONTENT -->
-
-    {% block footer %}
-        {% include 'templates/footer.html' %}
-    {% endblock footer %}
-    </div><!-- END .sliding-nav_page -->
-</div><!-- END .sliding-nav -->
+{% block header %}
+    {% include 'templates/header.html' %}
+{% endblock header %}
+
+<!-- PRIMARY CONTENT -->
+{% block content %}
+    This will be replaced in templates that extend this template
```

Comment:
It's pre-existing, there's a closing one as well (there's also comments for nearly every chunk of code in base.html). Going to leave this for now, we can discuss all code comments later.